stdlib/public/Concurrency/TaskGroup.swift

```diff
@@ -220,21 +215,15 @@ public struct TaskGroup<ChildTaskResult: Sendable> {
     }
   }
 
-  /// Add a child task to the group.
-  ///
-  /// ### Error handling
-  /// Operations are allowed to `throw`, in which case the `try await next()`
-  /// invocation corresponding to the failed task will re-throw the given task.
-  ///
-  /// The `add` function will never (re-)throw errors from the `operation`.
-  /// Instead, the corresponding `next()` call will throw the error when necessary.
+  /// Adds a child task to the group.
```

Comment:
Maybe 

```suggestion
  /// Spawns a child task in the group.
```

? We intend to familiarize developers with the "spawn" word

Reply:
Per editorial guidelines, I'm avoiding just repeating the same verb from a method name in its abstract when possible.  Using a different English word to describe what happens provides more information.